### PR TITLE
Add keyword support to SimpleAtsit5()

### LINKED ISSUE #11: Hardcode qold, gamma,.... etc. besides atol/rtol ?
Hey, I am making the final steps necessary to make this usable in **DynamicalSystems.jl**: using some internal norm as an adjustable field and allowing `init` to also take keywords.

At the moment `__init` doesn't take any keywords, they are all hardcoded to the integrator. So why make them a field of the integrator and not constants instead? Or something similar as in the `rs`, we put them all in a vector?

But abstol/reltol should be made adjustable through init (this is what I am doing now). What do you thing about the `gamma, etc.` ?

### PR DESCRIPTION
**Pull request by Datseris**
This PR makes all adaptive PI stepping parameters constants, besides abs/rel tol. They are instead made keyword arguments for the integrator.

In addition, one more keyword argument (the internalnorm) is added.

Closes #11

### CODE REVIEW

**Comment by Datseris** [comment]
@ChrisRackauckas This is ready for review!

**Comment by Datseris** [comment]
Ugh... The error is coming from the non-adaptive version ,which this PR didn't touch...

File: test/simpleatsit5_tests.jl [hunk 1]
```diff
@@ -40,3 +40,42 @@ step!(deiip); step!(deiip)
...
 
 sol = solve(odeoop,SimpleATsit5(),dt=dt)
+
+# Test keywords:
+oop = init(odeoop,SimpleATsit5(),dt=dt, reltol = 1e-9, abstol = 1e-9)
+step!(oop); step!(oop)
+deoop = DiffEqBase.init(odeoop, Tsit5(); dt = dt, reltol=1e-9, abstol=1e-9)
+step!(deoop); step!(deoop)
+
+@test oop.u ≈ deoop.u atol=1e-14
```
Comment (ChrisRackauckas) [inline test/simpleatsit5_tests.jl]: e-14 isn't stable across architectures

Comment (Datseris) [reply test/simpleatsit5_tests.jl]: Sure, I'll change this, but the problem is not with the ATsit, it is with the Tsit. This is where the error happens. And the error is big, like you need `atol = 1` to match the states...

Comment (ChrisRackauckas) [reply test/simpleatsit5_tests.jl]: Interesting, I swear it was passing locally?

Comment (Datseris) [reply test/simpleatsit5_tests.jl]: I was also super confused... :( 

I've changed the tolerances to 1e-9 everywhere to account for what you said.

**Pull request merged**